### PR TITLE
Added a book resource to prog-books-subject-en about JPEG (Computer Graphics)

### PR DESCRIPTION
**Pull request by pritammukherjee02**
## What does this PR do?
Add resource

## For resources
Programming Books by Subject (EN)

### Description
Added a new section for Computer Graphics and added a resource (Interactive Book link) for JPEG format in the same.

### Why is this valuable (or not)?
The PR will add a brand new section in the repo about a significantly large domain of computer science, i.e. Computer graphics. And a book discussing the most popular topic in the domain (JPEGs) has been added to assist those looking into Computer Graphics

### How do we know it's really free?
The link is to a Wikibooks (Wikimedia commons) resource, hence it is free.

### For book lists, is it a book? For course lists, is it a course? etc.
It is an interactive BOOK

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-subjects.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [Cloud Computing](#cloud-computing)
 * [Competitive Programming](#competitive-programming)
 * [Compiler Design](#compiler-design)
+* [Compiler Graphics](#compiler-graphics)
```
Comment (eshellman) [inline books/free-programming-books-subjects.md:17]: 1. please check your work
2. Why not use "graphics programming'?

Comment (pritammukherjee02) [reply books/free-programming-books-subjects.md:17]: 1. Hi. I'm sorry but I don't understand. Check my work as in?
2. Graphics Programming is a domain where people "code" or create things graphically. Like Scratch kind of is. Another example will be a no-code game engine. JPEG and all its concepts are about how to efficiently store, display and transmit visual information in form of images. the domain "Computer Graphics" suits it a lot more.

Here's an article showing what Graphics Programming is: https://www.finsliqblog.com/programming-languages/what-is-graphics-programming/#:~:text=Graphic%20or%20visual%20programming%20is,arrangement%20of%20the%20visual%20objects.

Thanks (I am new to Contributing to Open Source so I may make many mistakes haha please bear with me)

Comment (eshellman) [reply books/free-programming-books-subjects.md:17]: because we already have a "graphics programming" section.

**Comment by eshellman** [comment]
added in #9177

**Pull request closed without merging**